tests/testthat/test-lancelot-support.R

```diff
@@ -678,7 +700,11 @@ test_that("lancelot_compare combines likelihood correctly", {
   nms_pillar2_50_64 <- c("pillar2_50_64_pos", "pillar2_50_64_tot")
   nms_pillar2_65_79 <- c("pillar2_65_79_pos", "pillar2_65_79_tot")
   nms_pillar2_80_plus <- c("pillar2_80_plus_pos", "pillar2_80_plus_tot")
-  nms_react <- c("react_pos", "react_tot")
+  nms_react <- c("react_pos", "react_tot", "react_5_24_pos", "react_5_24_tot",
```

Comment:
can you revert `nms_react` and then have `nms_react_5_24` etc (similar to pillar 2 above)

Reply:
Then will need to add those into the `parts` list below